Sources/Operations/SwiftMarkConvertToHTMLOperation.swift

```diff
@@ -11,7 +11,7 @@ import Foundation
 /// A `SwiftMarkToHTMLOperation` converts any valid *CommonMark* text to HTML. Use this class if you want to convert *CommonMark* text to HTML asynchronously.
 ///
 /// The blocks you assign to process the fetched records are executed serially on an internal queue managed by the operation. Your blocks must be capable of executing on a background thread, so any tasks that require access to the main thread must be redirected accordingly.
-public class SwiftMarkToHTMLOperation: SwiftMarkOperation {
+open class SwiftMarkToHTMLOperation: SwiftMarkOperation {
```

Comment:
Since every class members are `public` this class should be `public` instead of `open`.